trailblazer/server/api.py

```diff
@@ -55,22 +56,48 @@ def before_request():
 @blueprint.route("/analyses")
 def analyses():
     """Display analyses."""
-    per_page = int(request.args.get("per_page", 100))
+    per_page = int(request.args.get("pageSize", 100))
     page = int(request.args.get("page", 1))
+    search = request.args.get("query")
+    is_visible = bool(request.args.get("isVisible", False))
+    sort_field = request.args.get("sortField", "started_at")
+    sort_order = request.args.get("sortOrder")
+
     analyses: Query = store.get_analyses_query_by_search_term_and_is_visible(
-        search_term=request.args.get("query"),
-        is_visible=bool(request.args.get("is_visible")),
+        search_term=search,
+        is_visible=is_visible,
     )
 
+    if sort_field:
+        column = getattr(Analysis, sort_field)
+        order_function = asc if sort_order == "asc" else desc
+        analyses = analyses.order_by(order_function(column))
+
+    filter_criteria = []
+    for key in request.args:
+        if key.endswith("[]"):
+            filter_key = key[:-2]
+            values = request.args.getlist(key)
```

Comment:
I assume this is a list?
```suggestion
            values: list = request.args.getlist(key)
```

Reply:
Removed this code